pkg/client/typed/dynamic/dynamic_util.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"k8s.io/kubernetes/pkg/api/meta"
 	metav1 "k8s.io/kubernetes/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/apis/meta/v1/unstructured"
```

Comment:
I can do that in a follow up, agree that's better.  There are 5 or 6 more refactors coming after this, trying to space them out.